crates/jstzd/src/image.rs

```diff
@@ -0,0 +1,160 @@
+use async_trait::async_trait;
+use futures_util::stream::StreamExt;
+use std::{collections::HashMap, sync::Arc};
+
+use bollard::{
+    image::{CreateImageOptions, ListImagesOptions},
+    Docker,
+};
+
+#[async_trait]
+pub trait Image {
+    fn image_name(&self) -> &str;
+    fn image_tag(&self) -> &str {
+        "latest"
```

Comment:
Extract into local `const LATEST_TAG` then
```suggestion
        Self::LATEST_TAG
```